### PR TITLE
bpo-44106: Improve sqlite3 example database contents

### PR DESCRIPTION
**Pull request by erlend-aasland**
Use programming languages iso. politicians as example content.

https://bugs.python.org/issue44106

### CODE REVIEW

**Comment by erlend-aasland** [comment]
Ref. https://github.com/python/cpython/pull/25003#discussion_r613135200

File: Doc/includes/sqlite3/ctx_manager.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import sqlite3
 
 con = sqlite3.connect(":memory:")
-con.execute("create table person (id integer primary key, firstname varchar unique)")
+con.execute("create table lang (id integer primary key, firstname varchar unique)")
```
Comment (zware) [inline Doc/includes/sqlite3/ctx_manager.py]: This one is incomplete; `firstname` doesn't make sense for `lang` and `person` is still referenced below.

File: Doc/includes/sqlite3/execsql_fetchonerow.py [hunk 2]
```diff
@@ -3,17 +3,17 @@
...
-SELECT = "select name_last, age from people order by age, name_last"
+SELECT = "select name, first_appeared from people order by first_appeared, name"
 
 # 1. Iterate over the rows available from the cursor, unpacking the
-# resulting sequences to yield their elements (name_last, age):
+# resulting sequences to yield their elements (name, first_appeared):
 cur.execute(SELECT)
-for (name_last, age) in cur:
-    print('%s is %d years old.' % (name_last, age))
+for (name, first_appeared) in cur:
```
Comment (berkerpeksag) [inline Doc/includes/sqlite3/execsql_fetchonerow.py]: Nitpick: Can you drop the parens as well?

File: Doc/includes/sqlite3/createdb.py [hunk 3]
```diff
@@ -12,15 +12,15 @@
...
         (
-          name_last      varchar(20),
-          age            integer
+          name           varchar(20),
+          first_appeared integer
         )
         """)
 
-cur.execute("insert into people (name_last, age) values ('Yeltsin',   72)")
-cur.execute("insert into people (name_last, age) values ('Putin',     51)")
```
Comment (berkerpeksag) [inline Doc/includes/sqlite3/createdb.py:23]: I'm a bit surprised we haven't been called out by this so far :P

File: Doc/includes/sqlite3/simple_tableprinter.py [hunk 4]
```diff
@@ -1,8 +1,8 @@
 import sqlite3
 
 FIELD_MAX_WIDTH = 20
-TABLE_NAME = 'people'
-SELECT = 'select * from %s order by age, name_last' % TABLE_NAME
+TABLE_NAME = 'lang'
```
Comment (berkerpeksag) [inline Doc/includes/sqlite3/simple_tableprinter.py]: I think we can just pass the query into the execute() method. No reason to keep them as constants as far as I'm aware.

File: Misc/NEWS.d/next/Documentation/2021-05-11-00-54-14.bpo-44106.grCmuu.rst [hunk 5]
```diff
@@ -0,0 +1,2 @@
+In the :mod:`sqlite3` docs, use programming languages instead of politicians
```
Comment (berkerpeksag) [inline Misc/NEWS.d/next/Documentation/2021-05-11-00-54-14.bpo-44106.grCmuu.rst]: We can drop the NEWS entry. We usually need NEWS entries for more bigger documentation changes.

**Review by berkerpeksag — CHANGES_REQUESTED** [review]

File: Doc/includes/sqlite3/insert_more_people.py [hunk 6]
```diff
@@ -4,13 +4,13 @@
 
 cur = con.cursor()
```
Comment (zware) [inline Doc/includes/sqlite3/insert_more_people.py:6]: Name of the file is now inconsistent :)

File: Doc/includes/sqlite3/simple_tableprinter.py (hunk 4 above)
Comment (erlend-aasland) [reply Doc/includes/sqlite3/simple_tableprinter.py]: I agree! (I was just trying to stay ahead of your review: being conservative and strictly excluding out-of-scope changes 😅 )

File: Misc/NEWS.d/next/Documentation/2021-05-11-00-54-14.bpo-44106.grCmuu.rst (hunk 5 above)
Comment (erlend-aasland) [reply Misc/NEWS.d/next/Documentation/2021-05-11-00-54-14.bpo-44106.grCmuu.rst]: Got it!

File: Doc/includes/sqlite3/ctx_manager.py (hunk 1 above)
Comment (erlend-aasland) [reply Doc/includes/sqlite3/ctx_manager.py]: Thanks, I'll fix those!

File: Doc/includes/sqlite3/insert_more_people.py (hunk 6 above)
Comment (erlend-aasland) [reply Doc/includes/sqlite3/insert_more_people.py:6]: Hah, good catch! I didn't think of that 😅

Comment (erlend-aasland) [reply Doc/includes/sqlite3/insert_more_people.py:6]: BTW, this file is not referenced in the .rst file. We should purge unused files from Doc/includes/sqlite3. Separate PR?

Comment (zware) [reply Doc/includes/sqlite3/insert_more_people.py:6]: Or here; I'll leave that choice up to you and Berker :)

**Comment by erlend-aasland** [comment]
I have made the requested changes; please review again

File: Doc/includes/sqlite3/createdb.py [hunk 7]
```diff
@@ -12,15 +12,15 @@
...
+cur.execute("insert into lang (name, first_appeared) values ('Forth', 1970)")
```
Comment (Fidget-Spinner) [inline Doc/includes/sqlite3/createdb.py:22]: Off topic: Probably only the fourth time in my life I've heard of Forth 😉.

**Review by Fidget-Spinner — APPROVED** [review]
Thanks! I learnt something new about programming languages while learning about sqllite :).

File: Doc/includes/sqlite3/createdb.py (hunk 7 above)
Comment (erlend-aasland) [reply Doc/includes/sqlite3/createdb.py:22]: Forth is a very interesting language! The bootloader on older Apple computers used to run a kind of a Forth interpreter. Great fun :)

Comment (erlend-aasland) [reply Doc/includes/sqlite3/createdb.py:22]: https://en.wikipedia.org/wiki/Open_Firmware

File: Doc/includes/sqlite3/insert_more_people.py (hunk 6 above)
Comment (berkerpeksag) [reply Doc/includes/sqlite3/insert_more_people.py:6]: We can remove stale examples in a later PR and use the same BPO issue.

Comment (erlend-aasland) [reply Doc/includes/sqlite3/insert_more_people.py:6]: All right, I'll prepare a PR. Thanks, Berker!

**Review by berkerpeksag — APPROVED** [review]

File: Doc/includes/sqlite3/insert_more_people.py (hunk 6 above)
Comment (erlend-aasland) [reply Doc/includes/sqlite3/insert_more_people.py:6]: Purge PR prepared: #26234

**Pull request merged**